logging/bin/change_internal_password.sh

```diff
@@ -249,7 +249,7 @@ if [ "$success" == "true" ]; then
           log_notice " the Fluent Bit pods to ensure log collection is not interrupted.            "
           log_notice "                                                                             "
           log_notice " This can be done by submitting the following command:                       "
-          log_notice "         kubectl -n $LOG_NS delete pods -l 'app=fluent-bit, fbout=es'        "
+          log_notice "       kubectl -n logging delete pods -l 'app.kubernetes.io/name=fluent-bit, fbout=es'"
```

Comment:
- Hard coded namespace name!
- Is there a syntax for filtering using the old and new labels?  I don't recall when the label changed but I suspect it is possible that someone is running an older version (using the app=fluent-bit label) when they run this.  **NEVER MIND...see next bullet**
- Killing the pods to get them to restart was/is a hack.  Let's use the `rollout resetart` command; it's cleaner and not dependent on pod labels.  The command to show is:  `kubectl -n logging rollout restart daemonset v4m-fb`
- (Of course, we don't want to hard-code the namespace name in that command)